pubspec.yaml

```diff
@@ -11,7 +11,7 @@ dependencies:
   args: '>=0.12.1 <2.0.0'
   coverage: ^0.12.0
   crypto: ^2.0.1
-  file: '>=2.3.7 <6.0.0'
+  file: '>=2.3.7'
```

Comment:
This seems VERY dangerous. We cannot sign-up to support every version of `pkg:file` forever?

Reply:
I'm following the lead of `process`, another `file` consumer and maintained by the same authors.

https://github.com/google/process.dart/blob/a5b677cb0799ee5e4fb4abf0ce48ec7e8b1f0ef5/pubspec.yaml#L10

(And actually more restrictive 🙄)

That said, I'm totally happy to tighten. 

@tvolkert, @srawlins: thoughts?

Also note that since this package is global activated as a rule (and not imported), I think this may be kind of academic.  Or?

Reply:
Just off chat w/ @tvolkert and will bump to `^5.0.0` 👍